README.md

```diff
@@ -50,7 +50,7 @@ Join the discussion @ [Discord](https://discord.gg/cG8JwrJB6Z)
 ### Invasion icons
   - invasion: `<grunt id>.png`
 ### Pokestop icons
-  - pokestop: `<lure id**>[_i][_q].png` ( `_i` invasion active `_q` quest active ) images from invasion and reward folder can be used as overlay
+  - pokestop: `0_l<lure id**>[_i][_q].png` ( `_i` invasion active `_q` quest active ) images from invasion and reward folder can be used as overlay
```

Comment:
that seems weird, didnt you mean didnt you mean `0[_l{lure id}][_i][_q].png` ?

Reply:
you're right. fixed it.